pyproject.toml

```
@@ -40,6 +40,7 @@ dynamic = ["version"]
 
 [project.scripts]
 beamlime = "beamlime.__main__:main"
+show-detector = "beamlime.executables.show_detector:main"
```

```suggestion
beamlime-raw-detectors = "beamlime.executables.show_detector:main"
```
To make it easier to discover, e.g., by tab-completion?